src/plugin/code.js

```diff
@@ -44,6 +124,20 @@ figma.ui.onmessage = async message => {
   }
 
   if (message.type === "POST_DIFFING_RESULT") {
-    const differences = message.content;
+    const { differences, modifiedFrames } = message.content;
+
+    differenceRectangleIdList.length = 0;
+
+    renderDifferenceRectangle(differences, modifiedFrames);
   }
 };
+
+figma.on("close", () => {
+  differenceRectangleIdList.forEach(rectangleId => {
+    const rectangleNode = figma.getNodeById(rectangleId);
```

Comment:
P3: `rectangleNode`라고 변수들이 되어있어서 `rectangleNodeId`라는 매개변수명을 사용해주어도 좋을 것 같습니다:)

```suggestion
  differenceRectangleIdList.forEach(rectangleNodeId => {
    const rectangleNode = figma.getNodeById(rectangleNodeId);
```

Reply:
`rectangleNodeId`가 더 직관적인 것 같네요!
수정하겠습니다 👍